src/style.css

```
@@ -1 +1,70 @@
-/* Styles go here */
+.header {
+  display: flex;
+  align-items: center;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  padding-left: 50px;
+  padding-right: 40px;
+  justify-content: space-between;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.header div > a {
+  display: flex;
+}
+
+.nav_list > li > a {
+  text-decoration: none;
+  font-size: 12px;
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  color: black;
+  display: inline-block;
+  padding: 22.5px 0;
+  text-transform: uppercase;
+}
+
+.nav_list li {
+  margin: 0 10px;
+}
+
+.nav_list {
+  list-style: none;
+  display: flex;
+  margin: 0;
+  padding: 0;
+}
+
+.nav_list:last-child {
+  margin-right: 0;
+}
+
+.nav_list > li > a:hover {
```

Не уведичивай специфичность
```suggestion
.link:hover {
```